changelog.txt

```diff
@@ -1,6 +1,7 @@
 *** Changelog ***
 
 = 9.0.0 - xxxx-xx-xx =
+* Fix - Allow account creation on checkout for subscriptions if enabled in settings.
```

Comment:
Should we mention that this change is specific to ECE in the changelog?

Reply:
Updating!